engine/src/flutter/lib/web_ui/lib/src/engine/semantics/menu.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'label_and_value.dart';
+import 'semantics.dart';
+
+/// Indicates a combo box element.
+///
+/// Uses aria combobox role to convey this semantic information to the element.
+///
+/// Screen-readers takes advantage of "aria-label" to describe the visual.
+class SemanticComboBox extends SemanticRole {
```

Comment:
Is the file name `menu.dart` correct here? Or should it be something like `combo_box.dart`?

Reply:
Ah, I was thinking to put all the menu related things together because our dropdown menu is also part of menu systems, so I named it menu.dart😀

Reply:
Consider rename to `menus`